omnibus/package-scripts/agent-dmg/postinst

```diff
@@ -57,11 +57,11 @@ chmod -v 755 "$RUN_DIR"
 echo "# Copying conf"
 mkdir -vp $CONF_DIR/checks.d
 
-if [ -e "/tmp/datadog.conf" ] || [ -e "/tmp/datadog.yaml" ]; then
+if [ -e "$INSTALL_DIR/tmp/datadog.conf" ] || [ -e "$INSTALL_DIR/tmp/datadog.yaml" ]; then
     mv -vf /tmp/datadog.conf /tmp/datadog.yaml $CONF_DIR
```

Comment:
❓ Question :
Shouldn't we move this one as well from `$INSTALL_DIR/tmp/datadog.conf` ?

Reply:
Yes you are right, I forgot this one. Thanks!